src/app/student/student.component.css

```diff
@@ -1,58 +1,290 @@
-.body {
-  margin-top: 50px;
-  color: #404F57;
+@media only screen and (max-width: 420px) {
+
+  .top {
+    margin: 5%;
+    margin-bottom: 0;
+    width: 90%;
+    height: 20%;
+  }
+  
+  .userinfo {
+    float: left;
+    text-align: right;
+  }
+  
+  .username {
+    color: #37474f;
+    font-size: 20px;
+    font-weight: bold;
+  }
+  
+  .location {
+    color: #455a64;
+    font-size: 15px;
+    font-weight: 500;
+  }
+  
+  .cert_info {
+    float: right;
+    height: 100%;
+  }
+  
+  .certy {
```

Comment:
Let's be honest we need a bit formal class name here. :laughing:

Reply:
@aashutoshrathi ran out of names 😄

Reply:
@aashutoshrathi done 😆